apps/common/react/components/Navbar/components/NavbarMenu.test.tsx

```diff
@@ -91,20 +91,10 @@ describe("NavbarMenu", () => {
       </MemoryRouter>
     );
 
-    const HomePage = screen.getByText("Strona główna");
-    const NewRepairPage = screen.getByText("Zleć naprawę");
-    const RepairStatusPage = screen.getByText("Stan naprawy");
-    const AboutUsPage = screen.getByText("O nas");
-    const GalleryPage = screen.getByText("Galeria");
-    const ContactPage = screen.getByText("Kontakt");
-    const Employee = screen.getByText("Pracownik");
+    menuItems.forEach(({ text }) => {
```

Comment:
No, previous version needs to come back. Typo in menuItems will result in the test passing, despite the typo